src/Geopilot.Api/Models/Mandate.cs

```diff
@@ -40,6 +40,24 @@ public class Mandate
     [NotMapped]
     public List<Coordinate> Coordinates { get; set; } = new List<Coordinate>();
 
+    /// <summary>
+    /// Defines how <see cref="Delivery.PrecursorDelivery"/> is evaluated.
+    /// </summary>
+    [Column(TypeName = "varchar(24)")]
+    public FieldEvaluationType EvaluatePrecursorDelivery { get; set; }
+
+    /// <summary>
+    /// Defines how <see cref="Delivery.Partial"/> is evaluated.
+    /// </summary>
+    [Column(TypeName = "varchar(24)")]
```

Comment:
Q: Weshalb `varchar(24)`?

Reply:
Für Enums haben wir dies bei geopilot bisher so gehandhabt, damit ein aussagekräftiger String in der DB steht.

Reply:
Das verstehe ich, jedoch ist die Länge von 24 etwas willkürlich. Da die erwarteten Werte bekannt sind, kann man die Länge entsprechend danach richten.

Reply:
✔️ : Also würdest du varchar(12) vorschlagen?